t/pod-spelling.t

```diff
@@ -55,6 +55,7 @@ mtu
 nullable
 ORed
 postgres
+PostgreSQL
```

Comment:
I don't see why this is necessary for this PR.

Reply:
It's not, it's a pre-existing failure that's causing `t/pod-spelling.t` to fail (something I have running now because of #766).

I'm happy to split it out into a separate PR if you'd like.

Reply:
Yes, this is a separate issue that I'd like to track down and resolve separately. It's not failing on master, obviously, so there is something else going on.

Reply:
Dropped in the latest rebase.